add script to generate geth genesis with preloaded contract accounts

jwasinger opened the pull request.

axic commented on genesis.py:
```diff
@@ -0,0 +1,34 @@
+
+  try:
+    with open(codefile, 'rb') as f:
+      genesis['alloc'][address] = binascii.hexlify(f.read())
```
This should also have the option to set initial balance.

Also would be nice to actually find and update an account entry if it exists.

jwasinger replied:
Done.

jwasinger commented:
Added geth genesis.  Changed preloaded accounts argument format to one json object, e.g. 

`python genesis.py --genesis geth-genesis.json --preload '{"0x00":{ "balance": "0x00", "code": "0xffff"}}'`

axic commented:
> Changed preloaded accounts argument format to one json object, e.g.

Does it mean that one would need to paste in 500k of hexstring on the CLI?

jwasinger commented:
No.  because the accounts take `codefile` as an argument.

axic commented:
@jwasinger what is the status of this?

axic commented:
@jwasinger what is the status of this?

jwasinger commented:
Irrelevant now.

jwasinger commented:
Let me save the branch
